src/Bicep.LangServer/IServiceCollectionExtensions.cs

```diff
@@ -39,4 +39,7 @@ public static class IServiceCollectionExtensions
 
     public static IServiceCollection AddBicepDecompiler(this IServiceCollection services) => services
         .AddSingleton<BicepDecompiler>();
```

Comment:
>BicepDecompiler [](http://example.com/codeflow?start=22&length=15)

Does anyone remember why we have this pattern for the decompiler even though it's just one interface that gets added to the DI container?